sql-native/src/main/java/org/ballerinalang/sql/utils/Utils.java

```diff
@@ -233,15 +232,9 @@ public static Object cleanUpConnection(BObject ballerinaObject, ResultSet result
 
     public static void updateProcedureCallExecutionResult(CallableStatement statement, BObject procedureCallResult)
             throws SQLException {
-        Object lastInsertedId = null;
         int count = statement.getUpdateCount();
-        ResultSet resultSet = statement.getGeneratedKeys();
-        if (resultSet.next()) {
-            lastInsertedId = getGeneratedKeys(resultSet);
-        }
         Map<String, Object> resultFields = new HashMap<>();
         resultFields.put(AFFECTED_ROW_COUNT_FIELD, count);
-        resultFields.put(LAST_INSERTED_ID_FIELD, lastInsertedId);
```

Comment:
Can you just assign a null value in the Procesure call rather not assigning it

Reply:
commited https://github.com/ballerina-platform/module-ballerina-sql/pull/258/commits/c485468dfe06f49dae978e6d697d5d85b8fabb9c